content/docs/studio/user-guide/team-collaboration/teams.md

```diff
@@ -96,22 +93,38 @@ even if the project belongs to a team where you are an `Editor` or `Admin`.
 | Manage columns                               | No      | No     | Yes    | Yes   |
 | Manage custom files                          | No      | No     | Yes    | Yes   |
 
+### Privileges in the team's model registry
```

Comment:
@jellebouwman @aguschin Can you pls review this table of access permissions on MR for team members, and comment if anything is missing or incorrect. Pls also review the note at the end (`\*All team members can submit model actions...`).